client/signup/steps/design-type/index.jsx

```diff
@@ -75,26 +74,22 @@ export class DesignTypeStep extends Component {
 		const choiceHandlers = this.getChoiceHandlers();
 
 		return (
-			<Card
-				key={ choice.type }
+			<Tile
+				buttonLabel={ choice.label }
+				description={ choice.description }
 				href={ `#${ choice.type }` }
```

Comment:
I believe this `href` is redundant and has the negative side effect of turning a `<div>` into an `<a href="#">`.

There is a `button` which handles focus nicely and keyboard activation nicely.
There is a `cursor:pointer` style on the `card.is-clickable`.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/24979/commits/a6b88c4afdd2de215903dd7c1b3d6128f40857b8 🔪

Reply:
This one wasn't removed:

https://github.com/Automattic/wp-calypso/blob/e2880839be3a88d2e1e30a3da4badffb28d61cba/client/signup/steps/design-type/index.jsx#L80

Reply:
Oh cmon, I'm sure I removed it. It must have slipped away...

Reply:
Great spot anyway @sirreal! Removed for real in #25250.